### PR TITLE
Add classic EAC and remove old Outlook on the web steps

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2850.

https://techcommunity.microsoft.com/t5/outlook-blog/faster-with-a-modern-design-and-new-features-the-new-outlook-on/ba-p/735222

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/mobile-access/remote-wipe-on-mobile-phone.md [hunk 1]
```diff
@@ -54,9 +54,14 @@ Since Outlook for iOS and Android appears as a single mobile device association
...
 > Having problems? Ask for help in the Exchange forums. Visit the forums at [Exchange Online](/answers/topics/office-exchange-server-itpro.html) or [Exchange Online Protection](https://social.technet.microsoft.com/forums/forefront/home?forum=FOPE).
 
-## Use the EAC to wipe a user's phone
+## Use the classic EAC to wipe a user's phone
 
-You can use the EAC to wipe a user's phone or cancel a remote wipe that has not yet completed.
+> [!NOTE]
+> The classic EAC will be fully deprecated by September 2022. For more information, see [Deprecation of the classic Exchange admin center in WW service](https://techcommunity.microsoft.com/t5/exchange-team-blog/deprecation-of-the-classic-exchange-admin-center-in-ww-service/ba-p/2736358).
+> 
+> This setting is not available on the new EAC. If the classic EAC was deprecated, you must use [Exchange Online PowerShell](#use-exchange-online-powershell-to-wipe-a-users-phone).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/mobile-access/remote-wipe-on-mobile-phone.md:62]:
```suggestion
> This setting is not available in the new EAC. If the classic EAC was deprecated, you must use [Exchange Online PowerShell](#use-exchange-online-powershell-to-wipe-a-users-phone).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**